http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/app/BaseAppInstallSpec.scala

```diff
@@ -50,7 +51,7 @@ class BaseAppInstallSpec extends AnyFlatSpecLike with LeonardoTestSuite with Moc
 
   val lzResources = LandingZoneResources(
     UUID.fromString("5c12f64b-f4ac-4be1-ae4a-4cace5de807d"),
-    AKSClusterName("cluster"),
+    AKSCluster("cluster", Map.empty[String, Boolean]),
```

Comment:
Mind explaining the `Map.empty` param?  I don't know Scala well enough to understand what this is for.

Reply:
Nevermind I think I understand it after re-reading.

My next question for the Scala people: might this be an appropriate place to use a default parameter value and just set this to empty automatically, if not provided?  Then you could just declare `AKSCluster("cluster")` throughout?

Reply:
You could certainly do that. However -- and this is just my personal preference -- I usually try to limit the use of default parameters. I find it makes the code harder to understand and easier to make mistakes, as opposed to being explicit. But 🤷